base/steer/FairRootManager.cxx

```diff
@@ -173,7 +173,7 @@ void FairRootManager::RegisterImpl(const char* name, const char* folderName, T*
         if (fSink) {
             fSink->RegisterImpl(name, folderName, obj);
         } else {
-            LOG(fatal) << "The sink does not exist to store persistent branches.";
+            LOG(error) << "The sink does not exist to store persistent branches.";
```

Comment:
Please excuse for noticing now.

The tutorial3 "test" helped!

```
[tut3-sampler][14:46:22.509645][ERROR][FairRootManager.cxx:176:RegisterImpl] The sink does not exist to store persistent branches.
```

Please consider outputting the `name` and/or `folderName` here as well (and that toFile is `true`)? That way people might be able to not call `Register` with `toFile=true`.